### ISSUE TITLE
Multiple D groups

### ISSUE THREAD

**Issue by funderburkjim** [funderburkjim/boesp-prep#20]
In several cases, a D-group has multiple entries.
This is a feature of the printed text. 
Not sure why these verses are presented differently in the text.
Not sure of ramifications, if any.
For reference, here are the instances:
```
2 D values ['<D145>', '<D146>']
3 D values ['<D295>', '<D296>', '<D297>']
2 D values ['<D580>', '<D581>']
2 D values ['<D988>', '<D989>']
2 D values ['<D1396>', '<D1397>']
2 D values ['<D2731>', '<D2732>']
2 D values ['<D2739>', '<D2740>']
3 D values ['<D2820>', '<D2821>', '<D2822>']
2 D values ['<D2912>', '<D2913>']
3 D values ['<D2961>', '<D2962>', '<D2963>']
2 D values ['<D4455>', '<D4456>']
2 D values ['<D7277>', '<D7278>']
```

**Comment by Andhrabharati** [comment]
> It is puzzling. There are a number of other fourliners that have only one D value.

@thomasincambodia 

pl. go through the https://github.com/funderburkjim/boesp-prep-ab/issues/1#issue-988559285 and the following posts therein.

**Comment by thomasincambodia** [comment]
@Andhrabharati 
The problem is I can't see why Boehtlingk has sometimes, if rarely, given 2 or 3  numbers 
to his translation and sometimes not.

284 =  4liner but 1 D there are quite a number of those

and then these:

\<D145\>\<D146\>\<A3408\> Hochmuth,·
\<D295\>\<D296\>\<D297\> Die Beschäftigungen·
\<D580\>\<D581\> Vorzügliche Menschen sind·
\<D988\>\<D989\>\<A372\>\<A373\> Ein Mann, der·
\<D1396\>\<D1397\>\<A3833\>\<A3834\> Wer anders·
\<D2731\>\<D2732\> Zehn Erzieher überragt·
\<D2739\>\<D2740\>\<A4160\>\<A4161\> Folgende·
\<D2820\>\<D2821\>\<D2822\>\<A4187\>\<A4188\>\<A4189\>·
\<D2912\>\<D2913\>\<A1216\>\<A1217\> Wenn ein·
\<D2961\>\<D2962\>\<D2963\>\<A1242\>\<A1243\>\<A1244\>·
\<D4455\>\<D4456\>\<A4985\>\<A4986\> Hat·
\<D7277\>\<D7278\>\<A5338\>\<A5339\> Ein Fürst·

perhaps there is a clue in the first edition of boesp

**Comment by Andhrabharati** [comment]
@thomasincambodia,

Probably you did not get the intent in my https://github.com/funderburkjim/boesp-prep-ab/issues/1#issue-988559285 posting.

Indian Prosodic works specify verses having a length of 1 to 32 syllables per line ('pāda' is the Skt. term); the syllables are defined for each differing pāda. The manuscripts have them as continuous text, from edge-to-edge on the material on which the texts are scribed/etched;  and the printed books have them arranged by lines as per some prescribed norms mostly the longer poem lines per one print line-- but to save print space, the shorter poem lines are clubbed together in one print line.

You can get a quick summary from Appendix-I (SANSKRIT PROSODY) in VS Apte's Practical Sanskrit-English Dictionary.
[This is available at Csl-doc (text form) and as a pdf (at download pages)]

There is no difference in the texts of 1st ed. and 2nd ed. of the Ind.Spr. volumes; taking for example "`<D988><D989><A372><A373> Ein Mann, der`"-

1st ed. has it as-
![image](https://user-images.githubusercontent.com/75209130/138078611-a530f777-be07-4540-a76e-2bcc3f615700.png)

and the 2nd ed. has it as-
![image](https://user-images.githubusercontent.com/75209130/138078875-5d8089e1-3933-4bd4-8a52-18d33e20ff2f.png)

In these, the first verse
    आयाति स्खलितैः पादैर्मुखवैवर्ण्यसंयुतः ।
    ललाटस्वेदभाग्भूरिगद्गदं भाषते वचः ॥
belongs to anuṣṭubh family (8 syllables per pāda), with the 
gggllggg    {m: 14}    [8: msgg]
llgglglg    {m: 12}    [8: srlg]
lggglggl    {m: 13}    [8: yrgl]
glgglglg    {m: 13}    [8: rrlg]

and the second verse
    अधोदृष्टिर्भवेत्कृत्वा पापं प्राप्तः सभां नरः ।
    तस्माद्यत्नात्परिज्ञेयश्चिह्नैरेतैर्विचक्षणैः ॥
also belongs to anuṣṭubh family  (8 syllables per pāda), with the
lggglggg    {m: 14}    [8: yrgg]
gggglglg    {m: 14}    [8: mrlg]
gggglggl    {m: 14}    [8: mrgl]
gggglglg    {m: 14}    [8: mrlg]
[Interestingly, this verse has a different text in the first quarter in both the editions (कम्पमानस्त्वधोऽवेक्षी vs. अधोदृष्टिर्भवेत्कृत्वा)!!]

Coming to D284-
    अनावर्ती कालो व्रजति स वृथा तन्न गणितं
	//दशास्तास्ताः सोढा व्यसनशतसंपातविधुराः ।
    कियद्वा वक्ष्यामः किमिव बत नात्मन्यपकृतं
	//वयं यावत्तावत्पुनरपि तदेव व्यवसितम् ॥
it belongs to śikhariṇī verse, having 17 syllables per line.
lggggglllllgglllg    {m: 25}    [17: ymnsBlg]
lggggglllllgglllg    {m: 25}    [17: ymnsBlg]
lggggglllllgglllg    {m: 25}    [17: ymnsBlg]
lggggglllllgglllg    {m: 25}    [17: ymnsBlg]

In summary, when Boehtlingk has to give one poem at once, it is rendered either as a 2-liner or a 4-liner, but with a single verse number; and when two or more poems are given together, they are given separate verse numbers (each being rendered either as a 2-liner or a 4-liner).

**Comment by Andhrabharati** [comment]
guess this issue is closable now.
